content/en/data_jobs/databricks.md

```diff
@@ -27,17 +28,60 @@ Follow these steps to enable Data Jobs Monitoring for Databricks.
    As an alternative, follow the [official Databricks documentation][10] to generate access token for a [service principal][11].
 
 1. In Datadog, open the Databricks integration tile.
-1. On the **Configure** tab, click **Add New**.
+1. On the **Configure** tab, click **Add Databricks Workspace**.
 1. Enter a workspace name, your Databricks workspace URL, and the Databricks token you generated.
-   {{< img src="data_jobs/databricks/configure-token.png" alt="In the Datadog-Databricks integration tile, a Databricks workspace is displayed. This workspace has a name, URL, and API token." style="width:100%;" >}}
+   {{< img src="data_jobs/databricks/configure-workspace-form.png" alt="In the Datadog-Databricks integration tile, a Databricks workspace is displayed. This workspace has a name, URL, and API token." style="width:100%;" >}}
+1. In the **Select products to set up integration** section, make sure the Data Jobs Monitoring product is **Enabled**.
+1. In the **Datadog Agent Setup** section, chooose to either
```

Comment:
I kept the links in the bold portion of the bullets, eg "Managed by Datadog" and "Manually" 

LMK if it should be different.

I removed global in this passage, it seems like a typo since the script is not global in that context

>Follow the instructions below[link] to install and manage the **~global~** init script for installing the agent globally or on specific Databricks clusters.

Reply:
Note the proposed UI changes for the modal 
<img width="2162" alt="config-products" src="https://github.com/user-attachments/assets/f4a1a4b8-539d-44d0-9c91-38207f5e4e94">